### PR TITLE
feat: products page

### PR DESCRIPTION
**Pull request by BelideNikhil**
added products page to the e-comm

image

### CODE REVIEW

**Comment by BelideNikhil** [comment]
Deploy Preview: https://project-97x.netlify.app/pages/product-page/product-page.html

**Review by someshtiwarii — APPROVED** [review]
looks really neat ! clean code.. good to merge

File: pages/product-page/product-page.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+                    Price Low to High
+                </label>
+                <label for="sort-by">
+                    <input type="radio" name="sort-by" id="">
+                    Price High to Low
+                </label>
+            </div>
+            <!--  -->
+            <hr>
+            <div class="filter-content">
```
Comment (kshitijthatte) [inline pages/product-page/product-page.html:59]: There's no styling with class name 'filter-content'

File: pages/product-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+}
+/* product section */
+.product-section {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+    grid-auto-rows: max-content;
+    gap: 1rem;
+}
+.product-section .card {
+    max-width: 340px;
```
Comment (kshitijthatte) [inline pages/product-page/product-page.css]: You can use rem unit here.

**Review by siddhigate — APPROVED** [review]
Clean code and UI! Good to merge👍

**Review by kshitijthatte — APPROVED** [review]

**Review by himadri2110 — APPROVED** [review]
Good to go!

**Pull request merged**